Leo-Chen-CM/ARGO-23
# Feat/mobile controls

I added mobile swipe support for our game. I also used the same parameters to allow you to play the game using your mouse to simulate mobile swipes for testing purposes. There is a build up on the google drive that will allow you test on your phone.

Steps to test:
- Run the game and use your mouse to swipe from left to right, player should move to the right
- Do the same but swap the inputs and player should move left
- Use your mouse to swipe from down to up and the player should jump
- do the same but swap the inputs and player should slide

There is an android build up on google drive so you can test the game on your phone also but the input commands are the exact same using mouse to simulate the mobile swipes so up to you really
